cs/src/comm/epoxy-transport/EpoxyListener.cs

```diff
@@ -176,5 +189,24 @@ private void ShutdownSocketSafe(Socket socket, EpoxyNetworkStream epoxyStream)
                 logger.Site().Debug(ex, "Exception encountered when shutting down a socket.");
             }
         }
+
+        private void LogClientStartResult(EndPoint remoteEndPoint, Task startTask)
+        {
+            if (startTask.IsFaulted)
+            {
+                logger.Site().Error(
+                    startTask.Exception,
+                    "Starting connection for {0} failed",
+                    remoteEndPoint);
+            }
+            else if (startTask.IsCanceled)
+            {
+                logger.Site().Debug("Starting connection for {0} was canceled", remoteEndPoint);
```

Comment:
My gut is telling me this should be a higher log level. I can't think of a time this would happen because everything's okay. Shutdown?